src/views/Home/index.tsx

```diff
@@ -133,7 +133,7 @@ const Home = ({ history }: HomeProps) => {
         {getSystemIntakeBanners()}
         {getBusinessCaseBanners()}
         <div className="tablet:grid-col-9">
-          <h1 className="margin-top-6">Welcome to EASi</h1>
+          <h1 className="margin-top-6">{t('general:welcome')}</h1>
```

Comment:
`general` is the locale namespace and is separated by `:`. From there, we can drill down the nested object with `.`.

For example:
```
const general = {
  home: {
    welcome: 'Welcome',
    description: 'Description'
  }
}

t('general:home.welcome')
// Welcome

t('general:home.description')
// Description
```